src/streamer/processors/asyncOpStream.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Org, StatusResult } from '@salesforce/core';
+import { Duration } from '@salesforce/kit';
+import { AnyJson, JsonMap, ensureJsonMap } from '@salesforce/ts-types';
+import { DOCeStreaming } from '../doceStream';
+
+export default class AsyncOpStreaming extends DOCeStreaming {
+  public constructor(org: Org, wait: Duration) {
+    super(org, wait);
+  }
+
+  /**
+   * It sends the correct config to the streamer
+   *
+   * @param asyncOpId the Id of the AOR we want to inspect
+   * @returns
+   */
+  public async startStreaming(asyncOpId: string): Promise<void | AnyJson> {
+    return this.startStream(
+      '/data/Async_Operation_Result__ChangeEvent',
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
+      this.startAsyncOpStreamProcessor.bind(this),
+      asyncOpId
+    );
+  }
+
+  /**
+   * This is the processor for the AOR CDC
+   *
+   * @param event The event JSON from the CDC
+   * @returns StatusResult
+   */
+  private startAsyncOpStreamProcessor(event: JsonMap): StatusResult {
+    const pl = ensureJsonMap(event.payload);
+
+    const c = ensureJsonMap(pl.ChangeEventHeader);
```

Comment:
This will be moved to the logger service once we have it right?

Reply:
What do you mean by this ???
The console.log ???

Reply:
Yes. We are printing the message before checking it's the recordId we are expecting.